shiny/api-examples/extended_task/app.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+from datetime import datetime
+
+from shiny import reactive, render
+from shiny.express import input, ui
+
+ui.h5("Current time")
+
+
+@render.text
+def current_time():
+    reactive.invalidate_later(1)
+    return datetime.now().strftime("%H:%M:%S")
+
+
+with ui.p():
+    "Notice that the time above updates every second, even if you click the button below."
+
+
+@reactive.extended_task
+async def slow_compute(a: int, b: int) -> int:
+    await asyncio.sleep(3)
+    return a + b
+
+
+with ui.layout_sidebar():
+    with ui.sidebar():
+        ui.input_numeric("x", "x", 1)
+        ui.input_numeric("y", "y", 2)
+        ui.input_task_button("btn", "Compute, slowly")
+        ui.input_action_button("btn_cancel", "Cancel")
+
+    slow_compute.bind_task_button("btn")
```

Comment:
Alternatively (see other comment about where `bind_task_button()` lives)

```suggestion
    ui.bind_task_button("btn", slow_compute)
```